src/components/tableview/TableTable.vue

```diff
@@ -152,6 +153,12 @@ export default {
     pendingEditList() {
       return Object.values(this.pendingEdits)
     },
+    pendingDeleteList() {
```

Comment:
Any reason to not combined edits and deletes into a `pendingOperations`?

Reply:
I did actually refactor it that way in preparation for implementing the add row functionality. I'll add the refactor part to this PR.